src/ImageData/FileLoader.cc

```diff
@@ -175,9 +175,11 @@ std::shared_ptr<casacore::CoordinateSystem> FileLoader::GetCoordinateSystem(cons
     return std::make_shared<casacore::CoordinateSystem>();
 }
 
-bool FileLoader::FindCoordinateAxes(casacore::IPosition& shape, int& spectral_axis, int& z_axis, int& stokes_axis, std::string& message) {
+bool FileLoader::FindCoordinateAxes(casacore::IPosition& shape, std::vector<int>& direction_axes, int& spectral_axis, int& stokes_axis,
```

Comment:
I would use "spatial" instead of "direction" in all these variables and comments, for consistency with our existing terminology.

Reply:
just a IIRC note, in casacore it is called directional axis. This may be why. 🙂

Reply:
If only we could all decide on one standard! :smile: